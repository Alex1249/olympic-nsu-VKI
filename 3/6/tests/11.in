15 2 20
0.1234567890abcde
