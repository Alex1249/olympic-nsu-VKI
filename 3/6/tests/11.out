0.00010011100101111
